cicecore/cicedyn/infrastructure/ice_domain.F90

```diff
@@ -81,6 +81,7 @@ module ice_domain
        distribution_wght     ! method for weighting work per block
                              ! 'block' = POP default configuration
                              ! 'blockall' = no land block elimination
+                             ! 'blockfull'= blockall but all blocks get full weight
```

Comment:
The comment here seems different from the description in the docs, which says 'minimal weight' rather than 'full weight'.  Or am I misunderstanding this?

Reply:
The documentation is updated.